cmake/configs/qurt_eagle_legacy_driver_default.cmake

```diff
@@ -64,10 +63,15 @@ set(config_module_list
 	# PX4 drivers
 	#
 	drivers/gps
-	drivers/uart_esc
 	drivers/qshell/qurt
 
 	#
+	# FC_ADDON drivers
+	#
+	platforms/qurt/fc_addon/rc_receiver
+	platforms/qurt/fc_addon/uart_esc
```

Comment:
I agree.  I would prefer to keep using the name uart_esc and that the newly added uart_esc driver use a different name.